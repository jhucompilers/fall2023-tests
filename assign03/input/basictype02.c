int main(void) {
  short s1;
  signed short s2;
  unsigned short s3;

  s1 = -1000;
  s2 = -1001;
  s3 = 30000;

  return 0;
}
