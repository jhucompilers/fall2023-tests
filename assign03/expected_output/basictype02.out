0|main|function|function () returning int
2|s1|variable|short
2|s2|variable|short
2|s3|variable|unsigned short
